packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -87,7 +86,7 @@ class Cell extends React.Component {
   /* eslint-enable class-methods-use-this */
 
   handleKeyDown(event) {
-    if (event.nativeEvent.keyCode === KEYCODES.ENTER || event.nativeEvent.keyCode === KEYCODES.SPACE) {
+    if (event.nativeEvent.keyCode === KeyCode.KEY_ENTER || event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
```

Comment:
For the KEYCODES.ENTER replacement, I think we actually want to use KEY_RETURN instead of KEY_ENTER. Our previous KEYCODES.ENTER value corresponds with the KEY_RETURN value. KEY_ENTER corresponds with the numpad 'Enter' key instead of the typical Return/Enter key we're expecting.

Reply:
For reference: 
https://github.com/kabirbaidhya/keycode-js/blob/master/lib/KeyCode.js#L6
https://keycode.info/

Reply:
Here is my fix. https://github.com/cerner/terra-clinical/pull/479/commits/e259d1518fbccecda91e49c423d8e346e437c924#diff-c5cd1aed974584ac171ba40b758f21e9R89